Agents/HeatEmissionAgent/README.md

```diff
@@ -1,54 +1,104 @@
+- Printing Plants: Average heat output per printer in units of megawatts. These values must be stored in a column called 'heat_emissions_per_printer'. The table must be called 'printing'.
+- Factories in industries other than precision engineering and printing: Specific energy consumption in units of MegaJoules per kilogram, production capacity in units of kilogram per second and thermal efficiency. Units other than kilograms can be used for the production volume as long as it is consistent between the specific energy consumption and production capacity. These values must be stored in columns called 'production_volume', 'specific_energy_consumption' and 'thermal_efficiency'. The table must be called 'factories'.
+- Individual Heat Sources in Chemical Plants and Petroleum Refineries: Carbon dioxide emissions in units of tons per year. These values must be stored in a column called 'co2_emissions'. The table name must be called 'jurong_island_city_furniture'.   
+
+
+
+
+##### Stack containers
+If the agent is being run as part of a stack, the user can opt to use a namespace located in the stack blazegraph. The procedure for spinning up the stack is described at [stack manager page](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager).
+
+#### 1.2 Docker Deployment
+
+- From the same directory location as this README, build the agent's image using the `docker compose build` command. If the container is run as part of a stack, copy the `heatemissionagent.json` file from the `stack-manager-input-config` folder into the `Deploy/stacks/dynamic/stack-manager/inputs/config/services` folder of the stack manager before starting the stack.
+
+
+### 2. Agent route
+
+The agent has a single API route which requires a POST request. It accepts the following input parameters:
+
+- ```dbName```: The name of the PostgreSQL database from which the factories' properties are queried. 
+
+The following is an example POST request for running the agent. The stack is assumed to have been spun up using the default port 3838:
+
+```
+curl -X POST -H "Content-Type: application/json" -d '{"dbName":"postgres"}'  "http://localhost:3838/heatemissionagent/performheatquery"
+```
+
+
+
+
+#### Return values 
+
+
+The agent adds a new column called 'heat_emissions' to each of the tables mentioned above containing the heat emissions of each type of industrial facility in units of megawatts.
+
+The following methods are used to estimate the heat emissions of each type of industrial facility:
+
+- Data Centres:
+
+$$ Q = \frac{1.08*P_{max}U_R + 21.53A_F + 9400}{1.0 \times 10^6} $$
+
+The parameters $Q$, $P_{max}$, $U_R$ and $A_F$ denote the heat emissions rate in megawatts, maximum IT capacity in megawatts, utilization rate and gross floor area in square meters respectively.
```

Comment:
Minor change - can you write out the parameters as a list rather than as a chunk of text, it is hard to follow

Reply:
Done.